README.md

```diff
@@ -107,7 +107,8 @@ setup scripts:
 
 ```console
 ./setup_miniconda.sh
-./setup_env.sh -n probtest -u
+source miniconda/bin/activate
+./setup_env.sh
```

Comment:
Maybe we could add a note saying something like: "For an unpinned environment setup use the `-u` option".

Reply:
I agree, I moved it into its own section, and make it a bit more verbose.

Reply:
Perfect, thank you!